amenity-points.mss

```diff
@@ -969,6 +970,15 @@
       marker-file: url('symbols/shop/variety_store.svg');
     }
   }
+  
+  [feature = 'advertising']{
+    [advertising = 'column'][zoom >= 19] {
```

Comment:
Currently we use single features like `[feature = 'advertising_column']`. It is a text concatenation in project.mml.